chirp/drivers/ft1d.py

```diff
@@ -1966,9 +1966,9 @@ def apply_WiresX_category(cls, setting, obj):
         val = setting.value.get_value()
         setattr(obj, "name", val)
 
-    def apply_WiresX_roomid(cls, setting, obj):
+    def apply_WiresX_roomid(self, setting, obj):
```

Comment:
Yeah, this (and others) should be `self` not `cls`.. Weird.

Reply:
I started down the path of fixing other instances but I realized some of the functions were modifying class-scoped variables, so in that aspect, it made sense logically. Plus it had nothing to do with the issue at hand so I only changed this one.

Reply:
Ack, there's almost no good reason for keeping per-class state so those are almost definitely bugs waiting to be found, but fair enough.